### PR TITLE
alpine 2.26

### PR DESCRIPTION
**Pull request by ellio167**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/alpine.rb [hunk 1]
```diff
@@ -31,6 +29,9 @@ class Alpine < Formula
     depends_on "linux-pam"
   end
 
+  # patch for macOS obtained from developer
+  patch :DATA
```
Comment (chenrui333) [inline Formula/alpine.rb:33]: Any upstream commit or issue reference for this?

**Comment by chenrui333** [comment]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, adhere to the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

File: Formula/alpine.rb [hunk 2]
```diff
@@ -1,11 +1,9 @@
 class Alpine < Formula
   desc "News and email agent"
   homepage "https://alpineapp.email"
-  url "https://alpineapp.email/alpine/release/src/alpine-2.25.tar.xz"
-  mirror "https://alpineapp.email/alpine/release/src/Old/alpine-2.25.tar.xz"
```
Comment (carlocab) [inline Formula/alpine.rb:5]: I don't think we want to remove this mirror. It doesn't work now, but it'll work when the next version is released and the `url` breaks.

File: Formula/alpine.rb (hunk 1 above)
Comment (ellio167) [reply Formula/alpine.rb:33]: I'll add one as soon as the developer commits these to the repo.

File: Formula/alpine.rb (hunk 2 above)
Comment (ellio167) [reply Formula/alpine.rb:5]: Ah, I see.  Sure, I'll put it back and add a comment explaining to ignore the corresponding `brew audit --strict --online` error...

File: Formula/alpine.rb [hunk 3]
```diff
@@ -31,6 +31,9 @@ class Alpine < Formula
     depends_on "linux-pam"
   end
 
+  # patch for macOS obtained from developer; see git commit 701aebc00aff0585ce6c96653714e4ba94834c9c
+  patch :DATA
```
Comment (carlocab) [inline Formula/alpine.rb:35]: Any reason why we're not using the patch at https://repo.or.cz/alpine.git/patch/701aebc00aff0585ce6c96653714e4ba94834c9c?hp=8b55be9f989f88aa0f464ec1c5f7dbef70310055?

Comment (carlocab) [inline Formula/alpine.rb:35]:
```suggestion
  # patch for macOS obtained from developer
  # https://repo.or.cz/alpine.git/commitdiff/701aebc00aff0585ce6c96653714e4ba94834c9c
  patch do
    url "https://repo.or.cz/alpine.git/patch/701aebc00aff0585ce6c96653714e4ba94834c9c"
    sha256 "fe0a41c025c6cd3e8eb3c51c9d0d96d2986eedd0d54ba9fbe29857a77853c0f4"
  end
```
Does this apply?

Comment (ellio167) [reply Formula/alpine.rb:35]: No, I'm afraid it fails; I guess there are other changes to the `configure` file in the history since the release of 2.26.

Comment (carlocab) [reply Formula/alpine.rb:35]: Hm, ok. Let's add a link directly to the upstream commit anyway to make it easier to find.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**